### PR TITLE
added search input component and hooked up main page search to store.…

### PR DESCRIPTION
**Pull request by srhboo**
… deleted sample code related to redux-observable setup (recipes)

### CODE REVIEW

**Comment by auredufour** [comment]
Good job Sarah, looks good to me! 👍

File: src/redux/reducers/index.js [hunk 1]
```diff
@@ -1,18 +1,23 @@
 import { combineReducers } from 'redux';
```
Comment (auredufour) [inline src/redux/reducers/index.js]: I don't think we should use the index to have any "cases" in it. The index should only contains all the imported reducers and combined them. We should create a reducer file for the different components, and then imported them to the index. How everyone feel about it?

Comment (srhboo) [inline src/redux/reducers/index.js]: Yes thanks for bringing it up! I agree, we should separate into files and use index to import all of them. I was unsure how to proceed with folder organization so I just left it in index. Either someone can merge the request and re-organize files or I can do it and just update this PR.

Comment (srhboo) [inline src/redux/reducers/index.js]: Updated - moved search reducer into separate file and imported into index.

**Comment by auredufour** [comment]
👍 look good, thank you Sarah for creating the search reducer!

**Pull request merged**